api/openapi.gen.json

```diff
@@ -101,7 +101,22 @@
         },
         "type": "object"
       },
-      "v1.InstanceType": {
+      "v1.AccountResponse": {
+        "properties": {
+          "account_number": {
+            "type": "string"
+          },
+          "id": {
+            "format": "int64",
+            "type": "integer"
+          },
+          "org_id": {
+            "type": "string"
+          }
+        },
+        "type": "object"
+      },
```

Comment:
Account Request and Response are unused, but I did not want to remove it in this PR. Can create another commit tho.

Reply:
Lets merge this and remove account endpoint in follow-up? Tho if you remove it in second commit, I'm also fine...